assets/js/app.js

```diff
@@ -1,3 +1,5 @@
+import 'babel-polyfill';
```

Comment:
I noticed that too.  I cherry-picked you commits over to my branch to create the bundles.  Guessing it will resolve itself when the PR is merged.